website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -1,18 +1,14 @@
 ## Spacing
 
-Spacing between sections, fields, text, and other form elements should follow a decreasing scale on an 8px grid. This creates a consistent vertical rhythm between elements and reinforces a natural hierarchy and association between elements.
+Spacing between sections, fields, text, and other form elements should follow a decreasing scale on an 8px grid. This creates a consistent visual hierarchy between elements.
 
 ### Form
 
-A **form** acts as a layout mechanism by wrapping the content, fields, and actions to apply consistent spacing while simultaneously handling the logic for submitting information and data collected by the fields.
-
-As a layout mechanism, a form consisting of more than one section should use a 32px gap in between sections.
-
-![Form container](/assets/patterns/form-patterns/form-container.png =450x*)
+A **form component** acts as a layout mechanism by wrapping the content, fields, and actions into sections with consistent spacing while simultaneously handling the logic for submitting information and data collected by the fields.
 
 ### Sections
 
-A **section** occupies the largest hierarchy within the form. Sections organize text, fields (inputs, checkboxes, toggles, etc.), and actions into logical sets. There should be a 32px vertical gap between each section within the form container.
+A **section** occupies the largest hierarchy within the form. Sections organize text, fields (inputs, checkboxes, toggles, etc.), and actions into logical groups. A form consisting of more than one section should use a 32px gap between sections.
```

Comment:
Removing "actions" because those are handled in the footer.

```suggestion
A section occupies the largest hierarchy within the form. Sections organize text and fields (inputs, checkboxes, toggles, etc.) into logical groups. A form consisting of more than one section should use a 32px gap between sections.
```